controllers/controllerNews.js

```diff
@@ -17,5 +17,16 @@ const controllerAddNews = async function (req, res, next) {
         throw new Error("something happened")
     }
 };
+const controllerFindById = async function (req, res, next){
+    try {
+        const id = req.params.id
+        const findNew = await serviceNews.FindById(id);
+        res.status(200).json(findNew);
```

Comment:
Si el id enviado no existe estas retornando un 200 con un body null. Para ese caso habria que retornar un 404 con un mensaje de not found.